docs/examples/modular.rst

```diff
@@ -4,10 +4,16 @@
 Modular Contracts
 *****************
 
-A modular approach to building your contracts helps you prevent overflow risks
-and unexpected tokens. In the example below, the contract uses the ``move`` method
+A modular approach to building your contracts helps you reduce the complexity.
+If you specify and control the behaviour or each module in isolation, the
+interactions you have to consider are only those between the specifications
+and not every other moving part of the contract.
+In the example below, the contract uses the ``move`` method
 of the ``Balances`` :ref:`library <libraries>` to check that balances sent between
```

Comment:
Not related to this PR: Could this and other references be reformaated as `hyperlink reference <URI>`_ so as to not interrupt the flow of text. Note that the `:ref: `library <libraries>` is rendered as-is.